keps/sig-testing/20190118-breaking-apart-the-kubernetes-test-tarball.md

```diff
@@ -0,0 +1,221 @@
+---
+kep-number: ?
+title: Breaking apart the Kubernetes test tarball
+authors:
+  - "@ixdy"
+owning-sig: sig-testing
+participating-sigs:
+  - sig-release
+  - sig-architecture
+reviewers:
+  - TBD
+approvers:
+  - TBD
```

Comment:
I'll be an approver

I'd like to suggest we get someone from https://github.com/kubernetes/sig-release/blob/master/release-engineering/OWNERS as an approver as well.  @tpepper? @hoegaarden?

Reply:
added you. not sure who else wants to jump in :)